pkg/envoy/registry/registry_benchmark.go

```diff
@@ -0,0 +1,66 @@
+package registry
```

Comment:
benchmark tests still belong int `*_test.go` files, so as not to be included in non test files